src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -3703,13 +3703,146 @@ _test_sample_versioned_api_example_4 (void)
    mongoc_server_api_destroy (server_api);
 }
 
+static int64_t iso_to_unix (const char* iso_str) {
+   /* TODO (CDRIVER-2945) there is no convenient helper for converting ISO8601
+    * strings to Unix timestamps. This is not shown in the example. */
+   return 1628330345;
+}
+
+static void _test_sample_versioned_api_example_5_6_7_8 (void) {
+#define N_DOCS 8
+   mongoc_client_t *client = NULL;
```

Comment:
nit: Some of the variables below are initialized, and others aren't. Especially since this is test code (not that that suggests a lower standard, but rather that it means there may be specific reasons to do certain things) I'm not against it, but it does feel inconsistent.

Reply:
True, there is no reason to initialize `client` or `server_api`, as they are set immediately after. I removed the initialization for consistency.